src/main/kotlin/net/atos/zac/app/planitems/model/RESTUserEventListenerData.kt

```diff
@@ -20,7 +20,7 @@ data class RESTUserEventListenerData(
 
     var resultaatToelichting: String? = null,
 
-    var resultaattypeUuid: UUID? = null,
+    var resultaattypeUuid: UUID,
```

Comment:
@hsiliev maybe you know better if this field should indeed be non-nullable? In the code I came across it was definitely non-nullable but not sure if I saw all occurrences so to speak.

Reply:
I'll leave it like this then. Should we get an issue with it, we know where to look. :-)